### DISCUSSION TITLE
想知道这个残缺不全的源码到底缺了多少东西...

### DISCUSSION THREAD

**Discussion by ruattd** [Meloong-Git/PCL#455] [提问] [answered]
这个源码在我本地的 VS 2022 上完全无法编译（当然这很正常）

但是不正常的是 VS 甚至报了 “无法将 FormMain 转换为 Window” 这种离谱的根本 Google 不到的错误...
看着源码我感觉它仿佛用了一个假 WPF ...

~~或者也可能我学的是假 VB 吧~~

**Comment by Pigeon0v0** [comment]
https://github.com/Hex-Dragon/PCL2/blob/main/.gitignore

看看这个呢

**Comment by LTCatt** [comment, marked as answer]
包含主题解锁、加密解密等保密代码
ModSecret.vb

设置存取系统，这包含了部分玩家账号信息
ModSetup.vb

包含主题解锁代码（关于页面、个性化设置页面）
PageOtherAbout.*
PageSetupUI.*

包含回声洞、开发者测试工具（百宝箱页面）
PageOtherTest.*

开发者工具
ModDevelop.vb

不公开底层函数库，以免抄袭
ModMain.vb
